examples/resources/service_authorization_basic_usage.tf

```diff
@@ -0,0 +1,11 @@
+resource "fastly_service_vcl" "myservice" {
```

Comment:
Can we add some example fields to each section, as this will make it more consistent with the other example files. For example, the [ACL Entries example](https://github.com/fastly/terraform-provider-fastly/blob/78fe6303e670d18a21884cabc610a93a9272a748/examples/resources/service_acl_entries_basic_usage.tf) provides configuration for all resources defined to make the example as complete as possible.

Reply:
I would also rename `"myservice"` to `"demo"` so the following `examples/resources/service_authorization_import.txt` file will align.

> **NOTE**: Our use of `myservice` and `demo` are a bit inconsistent, but looks like `demo` is used more.